gems/sorbet-runtime/.rubocop_todo.yml

```diff
@@ -167,6 +167,8 @@ Metrics/BlockLength:
 # Configuration parameters: CountComments, CountAsOne.
 Metrics/ClassLength:
   Max: 1098
+  Exclude:
+    - 'test/types/types.rb'
```

Comment:
fwiw I'm fine disabling these and similar cops

http://go/ps/blob/master/.rubocop.yml#L198-L214

^ Stripe's codebase disables them too